src/media/media-utils.ts

```diff
@@ -7,20 +9,26 @@ export let _hasTransceiverFunctionality: boolean;
 
 /**
  * Select or create the `audio.__gc-webrtc-inbound` element
+ * @deprecated use `createUniqueAudioMediaElement()` instead
  */
-export const getOrCreateAudioMediaElement = function (): HTMLAudioElement {
-  const existing = document.querySelector(`audio.${GC_AUDIO_EL_CLASS}`);
+export const getOrCreateAudioMediaElement = function (className: string = GC_AUDIO_EL_CLASS): HTMLAudioElement {
```

Comment:
I want to remove this method altogether. But since we do _technically_ export it, I figured we would deprecate it and remove it next major version bump.